tests/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  env: {
+    embertest: true
+  },
+  globals: {
+    t: true,
+    expectTranslation: true
```

Comment:
`true` means that test files can override the value, which seems odd. I would expect `false` for both of these.